compiler/src/dotty/tools/dotc/core/GadtConstraint.scala

```diff
@@ -253,7 +253,7 @@ final class ProperGadtConstraint private(
       case null => param
     }
 
-  private def tvarOrError(sym: Symbol)(using Context): TypeVar =
+  private def tvarOrThrow(sym: Symbol)(using Context): TypeVar =
```

Comment:
This is internal code. Maybe move this change to a separate PR.

Reply:
np. What would you think about a separate commit in the same PR?

Reply:
It is more about the discussion of the changes. The two changes share a name and nothing more. The one in Quotes will affect users and it would be better to have a PR that reflects that name change clearly.